features/testsuite.feature

```diff
@@ -62,6 +62,24 @@ Quentin check whether their testsuite works properly.
      When executing "go version" succeeds
      Then stdout should not be empty
 
+  @linux @darwin
+  Scenario: Able to use defined variable
+    When executing "VAR=$(echo 'hello')" succeeds
+     And executing "echo $VAR" succeeds
+    Then stdout should contain
+    """
+    hello
+    """
+
```

Comment:
`Then stdout should contain "hello"` may be enough? though most other tests are already using `"""`
I'm wondering whether `Scenario: Scenario Variables` isn't already testing something very similar?

Reply:
> seems like it has the same effect as When setting scenario variable "VAR" to the stdout from executing "echo 'hello'"

Effect is same but the underlying implementation is different in case of scenario variable it is using different function.